### PR TITLE
PAD bank account validation for premium accounts#4773

### PR DESCRIPTION
**Pull request by nitheesh-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/4773

*Description of changes:*
- PAD bank account validation for premium accounts#4773

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/store/modules/org.ts [hunk 1]
```diff
@@ -350,6 +352,14 @@ export default class OrgModule extends VuexModule {
     return response?.data
   }
 
+  @Action({ rawError: true })
+  public async validatePADInfo (): Promise<PADInfoValidation> {
+    const padInfo: PADInfo = { ...this.context.state['currentOrgPADInfo'] }
+    delete padInfo.isTOSAccepted
+    const response = await PaymentService.verifyPADInfo(padInfo)
```
Comment (saravanpa-aot) [inline auth-web/src/store/modules/org.ts:359]: no try catch here?

Comment (saravanpa-aot) [reply auth-web/src/store/modules/org.ts:359]: i saw that logic in other place..is it better if its handled here?

**Review by saravanpa-aot — COMMENTED** [review]
just about the createaccount in the catch..is there a way to move it out of catch?

**Comment by nitheesh-aot** [comment]
> just about the createaccount in the catch..is there a way to move it out of catch?

The there are multiple calls need to be done in the create account, it will be better to call the create account method in the catch in this case

**Review by saravanpa-aot — APPROVED** [review]
dicussed with nitheesh and suggested some options

**Pull request merged**